src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.Connections.cs

```diff
@@ -154,16 +157,16 @@ private static void ReportTimeout(Stopwatch watch)
 
             private static bool s_infoBarReported = false;
 
-            private static void ShowInfoBar()
+            private static void ShowInfoBar(Workspace workspace)
```

Comment:
Shouldn't we assert when this is null? The API is a no-op if this is null hence seems like a required paramter.

Reply:
I have no idea. I'm not familiar with this code and was simply trying to preserve behavior in the absence of static state class `PrimaryWorkspace`.